src/main/java/core/basesyntax/fao/FileWriterImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.fao;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements CustomFileWriter {
+    private static final String FILE_TO_WRITE = "finalReport.csv";
+
+    @Override
+    public void write(String info) {
+        String filePath = "output/" + FILE_TO_WRITE;
+
+        try {
+            Files.createDirectories(Paths.get("output"));
+        } catch (IOException e) {
+            throw new RuntimeException("Error creating output directory", e);
+        }
```

Comment:
```suggestion
```
Don’t complicate your solution. place file in the resources directory. Method  write to create the file if it doesn’t exist